README.md

```diff
@@ -19,3 +19,28 @@ Project members can directly update this repository.
 1. Create a feature branch using the following naming schema: *your_name*/*short_description*
 2. Push your branch to the repository and make a pull request to merge into the main branch
 3. Assign at least one reviewer and one person to handle the merge request. 
+
+## Setup docs and view documentation locally
+
+1. Create conda environment if desired:
+
+``` bash
+conda create --name IMPROVE python=3.9.16
```

Comment:
Will need to add the following to use the conda environment:
`conda activate IMPROVE`